test/1_stdlib/Tuple.swift

```diff
@@ -0,0 +1,110 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+import StdlibUnittest
+
+var TupleTestSuite = TestSuite("Tuple")
+
+// Test tuple comparison operators
+// all the tuple types use the same basic implementation for the operators
+// so testing any arity tests them all
+
+func testEquality<A: Equatable, B: Equatable, C: Equatable>(
```

Comment:
In the standard library code, we put spaces on both sides of the colon when it represents an 'is-a' relationship (in generic constraints, in inheritance and protocol implementation clauses etc.)